packages/gatsby-plugin-offline/README.md

```diff
@@ -166,6 +166,28 @@ const options = {
 }
 ```
 
+You can also use [Workbox CLI](https://developers.google.com/web/tools/workbox/modules/workbox-cli) to generate this file automatically. 
```

Comment:
If this is going to list the instructions for using workbox CLI perhaps it should include the npm install command from that link as well?